packages/vite/src/node/optimizer/scan.ts

```diff
@@ -466,7 +477,9 @@ async function transformGlob(
       start,
       normalizePath(importer),
       index,
-      root
+      root,
+      undefined,
```

Comment:
What is the point of this?

Reply:
because need to provide the next param `resolve`